### PR TITLE
Take monitoring error info URL from local_info.xml

### LINKED ISSUE #312: portal/GOCDB_monitor/check.php has a hard coded reference to goc.egi.eu
The page should reference the local_info.xml to determine the correct URL to display.

https://github.com/GOCDB/gocdb/blob/dev/htdocs/web_portal/GOCDB_monitor/check.php#L26

### PR DESCRIPTION
**Pull request by ineilson**
Fixes #312

### CODE REVIEW

File: htdocs/web_portal/GOCDB_monitor/check.php [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?php
 require_once "tests.php";
 
+// Initialise local_info.xml configuration overrides
+\Factory::getConfigService()->setLocalInfoOverride($_SERVER['SERVER_NAME']);
```
Comment (tofu-rocketry) [inline htdocs/web_portal/GOCDB_monitor/check.php:5]: The backslash is about namespacing, right? Is it needed here?

Comment (ineilson) [reply htdocs/web_portal/GOCDB_monitor/check.php:5]: Yes it's namespace spec and no, it's not needed here I think. However, as it is the most common usage in GOCDB codebase, I'll keep as is but make other references in this module follow the same.

**Review by gregcorbett — APPROVED** [review]
LGTM :)

**Comment by gregcorbett** [comment]
Just rebasing this so the test run against it.

**Pull request merged**